alerts/zookeeper/README.md

```diff
@@ -0,0 +1,39 @@
+# Alerts for Zookeeper in the Ops Agent
+
+## High Average Latency
+
+if `zookeeper.latency.avg` is 100ms or higher, it shows that the server can't keep up with demand and performance improvements should be sought after.
```

Comment:
Small nit: Please capitalize the first letter of the sentence in this description and below

Reply:
Updated